### DISCUSSION TITLE
Docker (QNAP Container Station) Install - Cannot mount shared folder

### DISCUSSION THREAD

**Discussion by Sar6e** [mealie-recipes/mealie#1767] [Q&A]
For better or worse, I'm using QNAP Container Station to run Docker Images.  I have successfully installed Mealie with one annoying exception.  I cannot get the image to mount /app/data to a shared folder on the host.  As a result, if I manually add images to recipes they are lost when I recreate the container.

Initially I though I must have just forgotten to make the shared drive mapping.  Last night I recreated the container, being absolutely certain I made the mapping.  Still nothing appearing in the shared folder when I added an image to the recipe.  This morning I thought permissions might be the problem so double checked the user/groups nominated in the PUID and PGID environment variables have read write access to the host volume and created the container the third time.  No luck.

I've got 8 other containers running and most have shared folder mappings to provide durable storage.  I just did a quick install of Kavita to make sure Container Station hasn't updated and 'broken' shared folders since the other containers were created.  It has two shared folder mappings and both work fine.

At this point, I'm stumped and would highly value any thoughts on what else to try.  Thanks!

**Comment by michael-genson** [comment]
Going to close this since it's been a while. If you're still having trouble, feel free to re-open
